docs/kafka/topic-configuration-kafka/README.adoc

```diff
@@ -301,7 +299,7 @@ h|Type
 |Integer
 
 h|Default value
-|`1 MB`
+|`1000000` (bytes, 1 MB)
```

Comment:
In the case of Trial Kafka Instances, default value is unlimited

Reply:
Not sure where @asanzgom got unlimited, but this is the default value for max message bytes that I can see
![image](https://user-images.githubusercontent.com/20477661/174616066-df8f6877-59da-424b-95c0-16f4faf9dec6.png)

Reply:
Perhaps he did not expand those collapsed lines and thought it was meant for Retention size.
![image](https://user-images.githubusercontent.com/20477661/174616767-6f64f385-c98f-4557-9440-316350fe0c43.png)

Reply:
@mmusil Ack.